contents/node/nodejs.md

```diff
@@ -135,7 +137,7 @@ When a dependency is installed, the package's code will be added to the local `/
 
 The [module section](#benefit-7-easy-to-reuse-code-from-others) describes how to import packages in your code.
 
-## Benefit 6: npm `scripts` is versatile and help to save time in the development process
+### Benefit 6: npm `scripts` is versatile and help to save time in the development process
```

Comment:
If the other section titles are not complete sentences, perhaps this title should follow suit?